### PR TITLE
Add support for PHP 7.2

### PR DESCRIPTION
**Pull request by thexpand**

### CODE REVIEW

**Comment by chris-kruining** [comment]
just a sec, I'll submit my fixes as a PR

**Comment by chris-kruining** [comment]
sorry, didn't notice this is a PR... whoops

**Comment by samsonasik** [comment]
php `7.1` and `7.2` need to be registered to .travis.yml

**Comment by thexpand** [comment]
@samsonasik Where can I take a look how it's done, so that I can complete this?

**Comment by samsonasik** [comment]
there is `.travis.yml` file which the php versions can be registered, for example https://github.com/zendframework/ZendDeveloperTools/blob/master/.travis.yml#L26-L35 , you can add php 7.1 and php 7.2 there after php 7

File: .travis.yml [hunk 1]
```diff
@@ -33,6 +33,26 @@ matrix:
     - php: 7
       env:
         - DEPS=latest
+    - php: 7.1
+      env:
+        - DEPS=lowest
+    - php: 7.1
+      env:
+        - DEPS=locked
+        - CS_CHECK=true
```
Comment (webimpress) [inline .travis.yml]: There is no need to run cs-check on every PHP version - please remove it here and also on PHP 7.2.

**Comment by thexpand** [comment]
@webimpress Is there anything else I can do to help?

**Comment by webimpress** [comment]
@thexpand It would be nice if you can update the .travis.yml configuration. So basicaly we need define `LEGACY_DEPS` (see for example: https://github.com/zendframework/zend-servicemanager/blob/master/.travis.yml#L22) - I can't say what it should be in LEGACY_DEPS list in case of this repo - you need to figure it out based on current build logs.
Then we need update these legacy deps, as it is here:
https://github.com/zendframework/zend-servicemanager/blob/master/.travis.yml#L62-L63

Then we should be able to update composer.lock to the latest version (by running `composer update`) and all build should be fine.

File: composer.json [hunk 2]
```diff
@@ -41,7 +41,7 @@
         "zendframework/zend-view": "^2.6"
     },
     "require-dev": {
-        "phpunit/phpunit": "^4.8 || ^5.0",
+        "phpunit/phpunit": "^5.7.25 || ^6.4.4",
```
Comment (webimpress) [inline composer.json]: We can have also `|| ^7.1`

**Comment by weierophinney** [comment]
Thanks, @thexpand. I have updated the Travis configuration, and also upped the minimum supported
version of PHPUnit so that it will work correctly with PHP 7.2; last tests passed all versions!

**Comment by thexpand** [comment]
@weierophinney Thanks for helping out. I was extremely busy lately and moreover I was not quite sure how to update the Travis file. Could you explain to me your changes?

**Comment by thexpand** [comment]
Detailed explanation as always :wink: Thank you once again, @weierophinney

**Pull request merged**